docs/content/howtos/_index.md

```diff
@@ -6,14 +6,6 @@ weight = 9
 
 ## How-tos
 
-This section includes LocalAI end-to-end examples, tutorial and how-tos curated by the community and maintained by [lunamidori5](https://github.com/lunamidori5).
+This section has moved to our maintainers site
```

Comment:
I think would make more sense to have a link to it in the README and a link in the menu (nearby the download) instead of keeping the page with a link to it

Reply:
I wasn't exactly sure how you wanted to do that sir. That sounds wonderful. Would you like me to edit it?